doc/dev/crimson/pipelines.rst

```diff
@@ -0,0 +1,279 @@
+=========
+Pipelines
+=========
+
+RADOS Protocol Requirements
+===========================
+
+A Ready-to-Go Checklist
+-----------------------
+
+In Ceph, librados I/O requests inquiry or mutate the shared state of the cluster.
+Because inherently a distributed storage system is distributed by its own nature, we can not assume
+that an OSD or the PGs hosted by it are always ready to serve a request targeting a shared state, not
+to mention the unpredictable hardware failures. But fortunately, even if an OSD is not able to serve
+a request right away, under most circumstances, it can postpone the request, prepare itself so that it
```

Comment:
s/the/

Reply:
Suggest:

"If an OSD is not able to serve a request right away, under most circumstances it can postpone and prepare to answer later, effectively:"